.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: required
+sudo: false
```

Comment:
But previously they had 
``` sh
sudo: required
```

I know default behaviour is container based but that also apply only if you have not specified any other behaviour(In our case we had it).